Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.2.1"
 [deps]
 ExprTools = "e2ba6199-217a-4e67-a87a-7c52f15ade04"
 IRTools = "7869d1d1-7146-5819-86e3-90919afe41df"
+LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
```

Comment:
If we only need `push_gate!` functions from ZXCalculus, shouldn't we only need the dependency of ZXCalculus here?

Reply:
I suggest we move what should be inside ZXCalculus first, or updating it would require a 2nd major version update.

Reply:
LightGraphs is needed because we need neighbors(zxd, v) here.